src/solver/solverUtils.F90

```diff
@@ -1193,7 +1295,9 @@ subroutine slipVelocitiesFineLevel_block(useOldCoor, t, sps)
           enddo
 
        enddo bocoLoop1
-
+#else
```

Comment:
do we usually add this `else continue` for `#ifndef`?

Reply:
the precompiler commands are removed in the end when it is compiled. does that UseOldCoor if check work w/o the continue line or does it trip?

Reply:
(I am not really getting the question sorry, just bumping this up)

Reply:
the `#ifdef` and `#endif` type statements are called precompiler directives I think. Those are the lines used to modify the actual code before it gets compiled. So if we are running a regular make w/o tapenade, and you want to exclude some lines, you do `#ifdef tapenade` or whatever the flag is, and because that flag will be false during a regular make, the precompiler will remove that chunk of the code. With that info, search a bit online to see how precompiler directives are actually used (my knowledge on this is tiny), and then these comments will make more sense. Then we can figure out whether we need them here or not.

Reply:
I am sorry, my reply was very general: I get the precompilers directives, and now I also understand which [if check](https://github.com/mdolab/adflow/blob/db808880b542f45a484365090ce6c97f7e8373df/src/solver/solverUtils.F90#L1146) you are referring to.. I just can't give you an answer, that's an old Mads' edit. Do you think is it worth to test this without the `else continue`?

Anyway, my question was more a general coding doubt, I think now I get the logic here, no need to change anything from my side.

Reply:
ah I see, it was for an empty if then statement. yeah its ok to keep I guess